packages/lesswrong/components/messaging/ConversationPage.tsx

```diff
@@ -34,6 +34,18 @@ const styles = (theme: ThemeType): JssStyles => ({
   }
 })
 
+const getTemplateHtml = ({html, displayName, greetingName}) => {
```

Comment:
Another :two-cent: nit, I would call this getDraftMessageHtml. It used to be a template, but now it's more of a draft. And there's an actual template that is taken as input, which is confusing.

Reply:
Yeah, sounds right.